jackal_description/urdf/accessories/pointgrey_bumblebee2_mount.urdf.xacro

```diff
@@ -0,0 +1,53 @@
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
```

Comment:
Lot of repetition between this file and the one below. Wondering about either a separate macro which spawns only the bracketry and not the camera that these two could wrap, or at least genericizing this to "camera_mount" with a parameter for which camera (flea3, bb2, etc).

Either way, we're getting into a splintering of files here. Consider amalgamating these into a single `camera_mount.urdf.xacro`, containing multiple macros as necessary.